pybites_tools/worldclock.py

```diff
@@ -28,13 +28,13 @@ def convert_time(hour: int = None, minute: int = None, tzone: str = None) -> Non
 
     for zone in timezones:
         try:
-            if not hour:
-                converted_time = datetime.now(pytz.timezone(zone))
-            else:
+            if hour in range(0, 23):
```

Comment:
23 is now included in the range. Thanks for pointing that out. I'll try to find some time next week for writing a test but I've not done any testing so far.

Reply:
@fischa I added a test here that will fail for this issue: https://github.com/PyBites-Open-Source/pybites-tools/pull/27/files

HTH
Bob

Reply:
Thx @bbelderbos - it looks like you've taken the changes of #26 and used them in #27 + the additional test you mentioned. Do you want to move forward with your PR? Then I can close this one.

In general - do you rather want to raise an issue first before getting a PR? I'm asking because there is no CONTRIBUTORS.md in the repo.

Reply:
PR is fine and yes we should get such an md in, thanks. I was actually going to close #27 so you can make the changes here. Another nice way to handle this I realized later is to validate the input args and raise an exception, I will shoot another demo PR and you can choose ...

Reply:
@fischa will you copy the test from #27 and commit it? Also I was thinking should we also be explicit about the minutes? E.g. `if hour in range(0, 24) and minute in range(0, 60):`

Reply:
@bbelderbos: Yes - I will pick that up shortly. Good catch with the minutes.